http-api/src/main/java/io/avaje/http/api/ValidationException.java

```diff
@@ -101,20 +92,5 @@ public String getField() {
     public String getMessage() {
       return message;
     }
-
```

Comment:
I originally left this mutable with getters/setters such that it would work out of the box with Jackson.

I'm thinking we can merge this in though and see how it goes.

Reply:
you only really need getters in that case